.github/workflows/multimodal-pr.yml

```diff
@@ -0,0 +1,178 @@
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: flake8
+        run: |
+          cd ${{ env.folder }}
+          make pip_test
+          make flake8
+
+  mypy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Set up Python ${{ env.pythonVersion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: mypy
+        run: |
+          cd ${{ env.folder }}
+          make pip_test
+          make mypy
+
+  pyright:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          environment-file: ${{ env.folder }}/environment.yml
+
+      - name: pyright
+        shell: bash -l {0}
+        run: |
+          conda info
+          cd ${{ env.folder }}
+          make pyright_install
+          make pyright
+
+  pytest:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Set up Python ${{ env.pythonVersion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: Install hi-ml-azure in editable mode
+        run: |
+          cd hi-ml-azure
+
+          # Install local package in editable mode
+          make pip_local
+
+      - name: Install hi-ml in editable mode
+        run: |
+          cd hi-ml
+
+          # Install local package in editable mode
+          make pip_local
+
+      - name: Test with pytest
+        run: |
+          cd ${{ env.folder }}
+
+          # Install local package in editable mode
+          make pip_local
+
+          # Run tests
+          make pip_test
+          make pytest
+
+      #- name: Run GPU tests
+      #  run: |
+      #    branch_prefix="refs/heads/"
+      #    full_branch_name=$GITHUB_REF
+      #    branch_name_without_prefix=${full_branch_name#$branch_prefix}
+      #    python hi-ml/run_pytest.py --mark=gpu --cluster=pr-gpu --conda_env=multimodal/environment.yml --folder=${{ env.folder }} --experiment="$branch_name_without_prefix"
+
+  build-python:
+    runs-on: ubuntu-18.04
+    needs: [ pytest ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          lfs: true
+
+      - name: Set up Python ${{ env.pythonVersion }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ env.pythonVersion }}
+
+      - name: Build Distribution
+        run: |
+          cd ${{ env.folder }}
+          make pip_build
+          make build
+
+      - name: Upload package artifacts
+        uses: ./.github/actions/upload_package_artifacts
+        with:
+          folder: ${{ env.folder }}
+
+  publish-pypi-pkg:
```

Comment:
will multimodal by published as a package to PyPI?

Reply:
I've removed publishing for now.